task/github.py

```diff
@@ -323,31 +323,24 @@ def pulls(self, state='open', since=None):
                 count += 1
         return result
 
-    # The since argument is seconds since the issue was either
-    # created (for open issues) or closed (for closed issues)
+    # The since argument is seconds since the issue was last time modified
     def issues(self, labels=["bot"], state="open", since=None):
         result = []
         page = 1
         count = 100
         label = ",".join(labels)
+        if since:
+            since = "&since={0}".format(time.strftime("%Y-%m-%dT%H:%M:%SZ", time.gmtime(since)))
```

Comment:
Does since= subsume the created_at/closed_at properties? I. e. these count as "modification" just like posting a comment or changing another issue attribute?

Reply:
yes, created_at and closes_at both count as the last modifications. You can see when issue was updated for example at  https://github.com/cockpit-project/cockpit/issues?q=is%3Aissue+sort%3Aupdated-desc